holidays/countries/hungary.py

```
@@ -117,6 +117,13 @@ class HungaryStaticHolidays:
     """Hungary special holidays.
 
     References:
+        * [2002](https://web.archive.org/web/20250526071659/https://jogkodex.hu/jsz/2001_25_gm_rendelet_9200619)
```

Pylint (3.3.7)

[convention] Line too long (112/100)

(C0301)